src/CustomOscillatorType.ts

```diff
@@ -1,3 +1,3 @@
-type CustomOscillatorType = 'sine' | 'square' | 'sawtooth' | 'triangle' | 'piano';
+type CustomOscillatorType = 'sine' | 'square' | 'sawtooth' | 'triangle' | 'piano' | 'betterpiano' | 'bestpiano' | 'guitar' ;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Garde que `betterpiano` et `guitar` into retire le vieux `piano`